device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -160,7 +160,7 @@ public Message(String body)
         initialize();
 
         this.body = body.getBytes(DEFAULT_IOTHUB_MESSAGE_CHARSET);
-        this.setContentType(DEFAULT_IOTHUB_MESSAGE_CHARSET.name());
+        this.setContentTypeFinal(DEFAULT_IOTHUB_MESSAGE_CHARSET.name());
```

Comment:
setContentType is deprecated method, this seemed to be a fine change. Let me know otherwise.

Reply:
Yep, this is a safe change